### ISSUE TITLE
bWeaponMatkRate

### ISSUE THREAD

**Issue by Felleonel** [rathena/rathena#2275]
* **rAthena Hash**: 9eaabd5

* **Client Date**:  2017-01-25RagexeRE

* **Server Mode**: Renewal

* **Description of Issue**: 
  * Result:  bWeaponMatkRate doesn't seem to be working when tested on Doram's Weapons (1692 and 1693). The difference of damage between a weapon with refine +7 and +10 is too few. 
  * How to Reproduce:  Test the damage of a Doram's Weapon (1692 or 1693), disabling and enabling the bonus bWeaponMatkRate from the item script.
   

* **Modifications that may affect results**:  None
  
  

Sorry for my bad english.

**Comment by secretdataz** [comment]
Can you try adding the modifier by a lot so that it become obvious that the bonus is working or not?

**Comment by Felleonel** [comment]
Yes, example, i changed ***bonus bWeaponMatkRate,40;*** for ***bonus bWeaponMatkRate,400;*** and the result was the same.
